src/pretix/api/urls.py

```diff
@@ -14,6 +14,7 @@
 
 event_router = routers.DefaultRouter()
 event_router.register(r'subevents', event.SubEventViewSet)
+event_router.register(r'clone', event.CloneEventViewSet)
```

Comment:
Just for the record, if I had implemented this, I'd probably used a [detail_route](http://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing) on the existing viewset instead of a new viewset, but I don't see any downsides to your approach, so I guess we keep it.

Reply:
Just gave the detail_route a quick test. There are some side effects regarding validation and responses that made it a bit tricky. So leaving as is for now.

Reply:
Alright, good to know!